pages/settings/wallets/SetupWallet.tsx

```diff
@@ -167,39 +160,17 @@ export function SetupWallet() {
               <XIcon className="text-foreground" />
             </TouchableOpacity>
           ) : (
-            <Dialog>
-              <DialogTrigger asChild>
-                <TouchableOpacity>
-                  <HelpCircleIcon className="text-foreground" />
-                </TouchableOpacity>
-              </DialogTrigger>
-              <DialogContent className="sm:max-w-[425px]">
-                <DialogHeader>
-                  <DialogTitle>Connect Your Wallet</DialogTitle>
-                  <View className="flex flex-col gap-2">
-                    <Text className="text-muted-foreground">
-                      Follow these steps to connect Alby Go to your Hub:
-                    </Text>
-                    <Text className="text-muted-foreground">
-                      1. Open your Alby Hub
-                    </Text>
-                    <Text className="text-muted-foreground">
-                      2. Go to App Store &raquo; Alby Go
-                    </Text>
-                    <Text className="text-muted-foreground">
-                      3. Scan the QR code with this app
-                    </Text>
-                  </View>
-                </DialogHeader>
-                <DialogFooter>
-                  <DialogClose asChild>
-                    <Button>
-                      <Text>OK</Text>
-                    </Button>
-                  </DialogClose>
-                </DialogFooter>
-              </DialogContent>
-            </Dialog>
+            <>
+              <TouchableOpacity onPressIn={() => setShowHelp(true)}>
+                <HelpCircleIcon className="text-foreground" />
+              </TouchableOpacity>
+              {showHelp && (
```

Comment:
Isn't the helpmodal shown / hidden via the visible property? Why do you still conditionally render the dialog? :thinking:

Reply:
Whoops, true, changed now, thanks!